cajita/src/CMakeLists.txt

```diff
@@ -36,25 +36,25 @@ if(HYPRE_FOUND)
     )
 endif()
 
-if(HEFFTE_FOUND)
+if(Heffte_FOUND)
   list(APPEND HEADERS_PUBLIC
     Cajita_FastFourierTransform.hpp
     )
 endif()
 
 add_library(Cajita ${SOURCES_PUBLIC})
 
-target_link_libraries(Cajita
+target_link_libraries(Cajita PUBLIC
   Kokkos::kokkos
   MPI::MPI_CXX
   )
 
 if(HYPRE_FOUND)
-  target_link_libraries(Cajita HYPRE::hypre)
+  target_link_libraries(Cajita PUBLIC HYPRE::hypre)
```

Comment:
Are we using hypre headers in our headers? Otherwise this should be `PRIVATE`.

Reply:
Yes - I use their headers in the linear solver implementation.